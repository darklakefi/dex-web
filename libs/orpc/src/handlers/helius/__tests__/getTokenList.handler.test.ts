import { getTokenListHandler } from "../getTokenList.handler";

describe("getTokenListHandler", () => {
  it("should return a list of assets", async () => {
    const result = await getTokenListHandler({
      limit: 10,
    });

    expect(result).toBeDefined();
    expect(result).toMatchSnapshot();
  });
});
