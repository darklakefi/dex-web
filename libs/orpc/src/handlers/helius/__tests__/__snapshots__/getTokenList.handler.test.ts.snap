// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getTokenListHandler > should return a list of assets 1`] = `
[
  {
    "description": "New range of formal shirts are designed keeping you in mind. With fits and styling that will make you stand apart",
    "id": "9e834d8e-58e5-4c10-adad-7426e48117f4",
    "image": null,
    "name": "Practical Wooden Ball",
    "symbol": "UNB",
  },
  {
    "description": "The Apollotech B340 is an affordable wireless mouse with reliable connectivity, 12 months battery life and modern design",
    "id": "dcaa170f-2189-4745-a6ce-d47a41ef27d0",
    "image": null,
    "name": "Licensed Concrete Fish",
    "symbol": "AWE",
  },
  {
    "description": "The Football Is Good For Training And Recreational Purposes",
    "id": "fd02d516-d3ad-4894-8cb6-932db5408464",
    "image": null,
    "name": "Refined Frozen Ball",
    "symbol": "TAS",
  },
  {
    "description": "The slim & simple Maple Gaming Keyboard from Dev Byte comes with a sleek body and 7- Color RGB LED Back-lighting for smart functionality",
    "id": "964d4e74-c94a-48db-8d72-70566658cb56",
    "image": null,
    "name": "Refined Fresh Shirt",
    "symbol": "RUS",
  },
  {
    "description": "Ergonomic executive chair upholstered in bonded black leather and PVC padded seat and back for all-day comfort and support",
    "id": "c4fd8f4b-d821-42a6-85e7-15f3ca0f1a2f",
    "image": null,
    "name": "Handcrafted Wooden Gloves",
    "symbol": "GEN",
  },
  {
    "description": "The slim & simple Maple Gaming Keyboard from Dev Byte comes with a sleek body and 7- Color RGB LED Back-lighting for smart functionality",
    "id": "a62dbbda-abef-4b73-a50e-6910dee76cbd",
    "image": null,
    "name": "Rustic Concrete Chips",
    "symbol": "LIC",
  },
  {
    "description": "The beautiful range of Apple Natural√© that has an exciting mix of natural ingredients. With the Goodness of 100% Natural Ingredients",
    "id": "2919c4bb-e5b1-4909-aec3-e5963c7fdfc9",
    "image": null,
    "name": "Generic Rubber Sausages",
    "symbol": "HAN",
  },
  {
    "description": "Carbonite web goalkeeper gloves are ergonomically designed to give easy fit",
    "id": "9692b271-f33a-4409-a78b-4432821048e4",
    "image": null,
    "name": "Sleek Wooden Bacon",
    "symbol": "AWE",
  },
  {
    "description": "Bostons most advanced compression wear technology increases muscle oxygenation, stabilizes active muscles",
    "id": "646815a5-dd44-4f9f-b722-ac277f139b0b",
    "image": null,
    "name": "Rustic Soft Ball",
    "symbol": "RUS",
  },
  {
    "description": "New range of formal shirts are designed keeping you in mind. With fits and styling that will make you stand apart",
    "id": "86974372-c6c7-4e65-8c79-3c23f92d3e7e",
    "image": null,
    "name": "Handmade Frozen Salad",
    "symbol": "INC",
  },
]
`;
